upgrading.adoc

```diff
@@ -501,23 +502,23 @@ way to give these tokens to your users. Again, short-notice. Right? No
 time to do this, has to be done quickly.
 ____
 
-He also lists one-time costs for merchants, payment processors,
-wallets, miners, and users. Then he talks about some permanent costs,
-for example higher risk of reorgs, privacy loss, etc.
+He also lists the one-time costs incurred by merchants, payment processors,
+wallets, miners, and users, as well as some of the permanent costs,
+for example privacy loss and a higher risk of reorgs.
 
-Also, when a split has happened, and the chain with the most general
+Indeed, when a split happens and the chain with the most general
 rules becomes stronger than the chain with the stricter rules, a reorg
 will occur. This will have a severe impact on all transactions carried
 out in the wiped-out branch. For these reasons it's really important
-to try to avoid splits at all times.
+to try avoiding chain splits at all times.
 
 === Conclusion
```

Comment:
Generally, I'm not sure we should keep the Conclusion sections in the chapters, because all chapters don't have them. We should either ditch them or write conclusions for the ones lacking.

What do you think. Are the conclusions helpful for the reader?

Reply:
I think introductions and conclusions always help the reader focus on the essential message, which is why they are always included in academic papers, for instance. I would try to add a small conclusion paragraph to each chapter, that summarizes the main concepts and the takeaway message, if there is one. Do you like the idea?